core/color/color_test.go

```diff
@@ -0,0 +1,43 @@
+package color
+
+import (
+	"testing"
+)
+
+func TestOpposite(t *testing.T) {
+	testCases := []struct {
+		desc string
+		in   string
+		want string
+	}{
+		{
+			desc: "white=>black",
+			in:   "W",
+			want: "B",
+		},
+		{
+			desc: "black=>white",
+			in:   "B",
+			want: "W",
+		},
+		{
+			desc: "empty=>empty",
+			in:   "",
+			want: "",
+		},
+		{
```

Comment:
Good to distiinguish between the test conditions of an empty string and a random string, even though both return the same general result; ie the same value as was entered, because the two inputs obviously have different data types.